apps/frontend/src/pages/projects/styles.module.css

```diff
@@ -17,3 +17,9 @@
 	gap: 12px;
 	padding: 20px 0;
 }
+
+.no-projects-message {
```

Comment:
I think the placeholder should be not so bright, also the font-size should be the same as for body, and it I think it will look better if it will be horizontally centered